### ISSUE TITLE
请问大佬是如何转模型的呢

### ISSUE THREAD

**Issue by Yulin-CC** [pengyang1225/FastestDet_NCNN#1]

**Comment by pengyang1225** [comment]
class DetectHead(nn.Module):
    def __init__(self, input_channels, category_num):
        super(DetectHead, self).__init__()
        self.conv1x1 =  Conv1x1(input_channels, input_channels)

        self.obj_layers = Head(input_channels, 1)
        self.reg_layers = Head(input_channels, 4)
        self.cls_layers = Head(input_channels, category_num)

        self.sigmoid = nn.Sigmoid()
        self.softmax = nn.Softmax(dim=1)
        self.export_onnx=False
        
    def forward(self, x):
        x = self.conv1x1(x)
        
        obj = self.sigmoid(self.obj_layers(x))
        reg = self.reg_layers(x)
        cls = self.softmax(self.cls_layers(x))

        print("export onnx ...")
        if self.export_onnx:
            return torch.cat((obj, reg, cls), dim =1).permute(0, 2, 3, 1)

        return torch.cat((obj, reg, cls), dim =1)
